cylc/flow/data_messages.proto

```diff
@@ -144,7 +144,6 @@ message PbJob {
     optional float execution_time_limit = 14;
     optional string platform = 15;
     optional string job_log_dir = 17;
-    repeated string extra_logs = 29;
```

Comment:
@dwsutherland 

This field is no longer needed, is it safe to remove a "repeated string"?

Could this cause compatibility problems between cylc-flow and cylc-uiserver at different versions?

Reply:
I think this is fine, since I don't think we ask for this field anywhere.. And as long as we don't reuse `29`..